hazelcast/src/test/java/com/hazelcast/cluster/JoinStressTest.java

```diff
@@ -199,4 +240,46 @@ private void initNetworkConfig(NetworkConfig networkConfig, int basePort, int po
         tcpIpConfig.setMembers(members);
     }
 
+    public class MemberInfoUpdateOperationSerializer implements StreamSerializer<MemberInfoUpdateOperation> {
+        @Override
+        public void write(ObjectDataOutput out, MemberInfoUpdateOperation object) throws IOException {
+            object.writeData(out);
+        }
+
+        @Override
+        public MemberInfoUpdateOperation read(ObjectDataInput in) throws IOException {
+            final DelayedMemberInfoUpdateOperation operation = new DelayedMemberInfoUpdateOperation();
+            operation.readData(in);
+            return operation;
+        }
+
+        @Override
+        public int getTypeId() {
+            return 9999;
+        }
+
+        @Override
+        public void destroy() {
+
+        }
+    }
+
+    public static class DelayedMemberInfoUpdateOperation extends MemberInfoUpdateOperation {
+
+        public DelayedMemberInfoUpdateOperation() {
+        }
+
+        @Override
```

Comment:
Why do you copy paste all this logic? You can just add a sleep and then call super.run().